frontend/src/views/GShootItemTerminal.vue

```diff
@@ -33,8 +48,19 @@ export default {
       terminalSplitpanes,
     }
   },
-  mounted () {
-    this.load()
```

Comment:
was this removed by accident? Now the create new terminal window does not open anymore